config/groups.yaml

```diff
@@ -1,3 +1,9 @@
+- name: "SSHB"
+members:
+    -"2632729"
+    -"2565267"
+    -"2595703"
+    -"2625424"
```

Comment:
This isn't valid YAML. See the other entires in the file for how to format this correctly.

Reply:
Hi we only saw this message yesterday and I sent another pull request with the correct syntax under my name instead, sorry for the delay in response time

Reply:
Thanks for letting me know. It would have been better to update this PR, rather than create a new one. I will go ahead and close this one then if it has been superseded by https://github.com/uob-sepp/groups-2024/pull/129